### ISSUE TITLE
Cargo build failed with rustc 1.26.0-nightly

### ISSUE THREAD

**Issue by rleungx** [JeffBelgum/ntp#12]
```
   Compiling time v0.1.39
   Compiling ntp v0.4.0 (file:///Users/.../Workspace/ntp)
error[E0034]: multiple applicable items in scope
  --> src/packet.rs:48:18
   |
48 |         let li = LeapIndicator::try_from(li_vn_mode >> 6)?;
   |                  ^^^^^^^^^^^^^^^^^^^^^^^ multiple `try_from` found
   |
   = note: candidate #1 is defined in an impl of the trait `std::convert::TryFrom` for the type `_`
note: candidate #2 is defined in the trait `conv::TryFrom`
  --> /Users/.../.cargo/registry/src/github.com-1ecc6299db9ec823/conv-0.3.3/src/lib.rs:377:5
   |
377|     fn try_from(src: Src) -> Result<Self, Self::Err>;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: to disambiguate the method call, write `conv::TryFrom::try_from(...)` instead

error[E0034]: multiple applicable items in scope
  --> src/packet.rs:49:18
   |
49 |         let vn = Version::try_from((li_vn_mode >> 3) & 0b111)?;
   |                  ^^^^^^^^^^^^^^^^^ multiple `try_from` found
   |
   = note: candidate #1 is defined in an impl of the trait `std::convert::TryFrom` for the type `_`
note: candidate #2 is defined in the trait `conv::TryFrom`
  --> /Users/.../.cargo/registry/src/github.com-1ecc6299db9ec823/conv-0.3.3/src/lib.rs:377:5
   |
377|     fn try_from(src: Src) -> Result<Self, Self::Err>;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: to disambiguate the method call, write `conv::TryFrom::try_from(...)` instead

error[E0034]: multiple applicable items in scope
  --> src/packet.rs:50:20
   |
50 |         let mode = Mode::try_from(li_vn_mode & 0b111)?;
   |                    ^^^^^^^^^^^^^^ multiple `try_from` found
   |
   = note: candidate #1 is defined in an impl of the trait `std::convert::TryFrom` for the type `_`
note: candidate #2 is defined in the trait `conv::TryFrom`
  --> /Users/.../.cargo/registry/src/github.com-1ecc6299db9ec823/conv-0.3.3/src/lib.rs:377:5
   |
377|     fn try_from(src: Src) -> Result<Self, Self::Err>;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: to disambiguate the method call, write `conv::TryFrom::try_from(...)` instead

error[E0034]: multiple applicable items in scope
  --> src/packet.rs:60:26
   |
60 |             let source = PrimarySource::try_from(ref_id_raw)?;
   |                          ^^^^^^^^^^^^^^^^^^^^^^^ multiple `try_from` found
   |
   = note: candidate #1 is defined in an impl of the trait `std::convert::TryFrom` for the type `_`
note: candidate #2 is defined in the trait `conv::TryFrom`
  --> /Users/.../.cargo/registry/src/github.com-1ecc6299db9ec823/conv-0.3.3/src/lib.rs:377:5
   |
377|     fn try_from(src: Src) -> Result<Self, Self::Err>;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: to disambiguate the method call, write `conv::TryFrom::try_from(...)` instead

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0034`.
error: Could not compile `ntp`.

To learn more, run the command again with --verbose.
```

**Comment by rleungx** [comment]
See rust-lang/rust#49305.

**Comment by rleungx** [comment]
Fixed by rust-lang/rust#49518.

**Comment by rleungx** [comment]
@JeffBelgum The change was reverted for now.
